Doppler.Sap/Services/BillingServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,37 @@
+using Doppler.Sap.Models;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Doppler.Sap.Services
+{
+    public static class ServicesCollectionExtensions
```

Comment:
The convention to create extensions to `IServiceCollection` is place them into the namespace `Microsoft.Extensions.DependencyInjection`:

> [...] We recommended that apps follow this naming convention. Place extension methods in the `Microsoft.Extensions.DependencyInjection` namespace to encapsulate groups of service registrations.
> - [Register groups of services with extension methods](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-3.1#register-groups-of-services-with-extension-methods)